.gitignore

```diff
@@ -13,3 +13,4 @@ tmp/
 .env
 coverage
 storybook-static/
+packages/icons/dist/
```

Comment:
For some reason, the files in this particular `dist/` dir weren't being ignored.

Reviewer: try removing this line on your machine, run `npm run build:svg`, and see if git ignores what gets created

Reply:
It doesn't for me.
![screenshot from 2019-01-11 09-16-02](https://user-images.githubusercontent.com/40791275/51018804-92d66780-1581-11e9-8958-fb2b6034123f.png)